### DISCUSSION TITLE
Sink/source terms in `openpnm`

### DISCUSSION THREAD

**Discussion by Mat10G** [PMEAL/OpenPNM#2493] [Q&A]
Hello everyone, I post this message because I have a query regarding the example "Diffusion with source and sink terms". I was wondering if the terms source and sink would refer only to chemical reactions (if so, which one can be modelled ? Dissolution, gazification, mineralization... ), or if it only refers to physics with, for example, continuity equation ? Thank you really much for your help and advices.

<img width="406" alt="Sink and source terms" src="https://user-images.githubusercontent.com/103643299/176894426-bd6f3283-d18e-4640-9a45-6556dcca5bd9.PNG">

**Comment by oscarhx** [comment]
Hello, I happened to work on it recently. If you track into the source you may find "power law" term indicates the rule how the reaction worked. to my knowledge, in your case, if sinkb=1, then, it indicate a linear reaction keneral that equals sinkA*pore.concentration. just like it, you should desgin your power law term to fit to your specific physical problem. ~~)

**Reply by Mat10G** [reply]
Hello, thank you very much for your answer
